### DISCUSSION TITLE
gRPC streaming calls

### DISCUSSION THREAD

**Discussion by maxwagner-dev** [roadrunner-server/roadrunner#1349] [Q&A] [answered]
Hello,

is it possible to have gRPC streaming calls (server streaming, client streaming, and bi-directional streaming) for roadrunner?
If yes, can you share an example?
If not, will it be implemented, and when?

Thank you very much.

**Comment by rustatian** [comment, marked as answer]
Hey @maxwagner-dev 👋🏻 

Unfortunately, this is not possible at the moment. We have this ticket to track the issue: https://github.com/roadrunner-server/roadrunner/issues/923

I guess, in the next release (`v2.13`), we will add at least response streaming support.

**Comment by rustatian** [comment]
We have plans to add request and response streams for the gRPC and HTTP (response) in 2.13 (~month from now).
